### PR TITLE
feat: add jax frontend function qdwh

### LINKED ISSUE #25633: qdwh

### PR DESCRIPTION
**Pull request by rukiyearican**
# PR Description

## Related Issue

 I need to define/use other functions too, however if i do that, the code will be very long. It did not feel right,  i'm not sure what should i do. I hope it's not an unprofessional thought
 
My unsure things: from jax._src import core / defining _mask() / defining _qdwh
Close #25633

## Checklist

- [x] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Please don't forget to add a test in `ivy\_tests/test\_ivy/test\_frontends/test\_jax/test\_jax\_lax\_linalg.py`

**Comment by Daniel4078** [comment]
you can find more info about how to write frontend functions here: https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#writing-frontend-functions

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 from hypothesis import strategies as st
 
 # local
-import ivy_tests.test_ivy.helpers as helpers
-from ivy_tests.test_ivy.helpers import assert_all_close
-from ivy_tests.test_ivy.helpers import handle_frontend_test, BackendHandler
+import ivy.ivy_tests.test_ivy.helpers as helpers
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:7]: Just curious, why did you make these changes?

Comment (Daniel4078) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:7]: it causes this error: E   ModuleNotFoundError: No module named 'ivy.ivy_tests'
https://github.com/unifyai/ivy/actions/runs/6371908597/job/17294346655?pr=26218#step:3:32

Comment (rukiyearican) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:7]: Hello, sorry for making that changes, i didnt mean to but this caused the same problem, i didnt know what to do; ModuleNotFoundError: No module named 'ivy_tests'

Also i really need some help, what should i do about defining _mask() / defining _qdwh()

Comment (Daniel4078) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:7]: Please check out other examples here: https://github.com/unifyai/ivy/blob/4186719e16321de3d7258080583b97b4ea8d9176/ivy_tests/test_ivy/test_frontends/test_jax/test_general_functions.py#L1

**Comment by Daniel4078** [comment]
By the way, please check out https://unify.ai/docs/ivy/overview/deep_dive/exception_handling.html to see if the import _core thing is necessary or not.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py (hunk 1 above)
Comment (Daniel4078) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:7]: > Hello, sorry for making that changes, i didnt mean to but this caused the same problem, i didnt know what to do; ModuleNotFoundError: No module named 'ivy_tests'
> 
> Also i really need some help, what should i do about defining _mask() / defining _qdwh()

I think you can compositionally utilize other ivy functions like what you have already done.

**Pull request closed without merging**